Mjml.Net/Components/Body/SectionComponent.cs

```diff
@@ -1,11 +1,595 @@
-﻿namespace Mjml.Net.Components.Body
+﻿using Mjml.Net.Extensions;
+
+namespace Mjml.Net.Components.Body
 {
     public partial class SectionComponent : BodyComponentBase
     {
         public override string ComponentName => "mj-section";
 
+        [Bind("background-color", BindType.Color)]
+        public string? BackgroundColor;
+
+        [Bind("background-position", BindType.String)]
+        public string BackgroundPosition = "top center";
+
+        [Bind("background-position-x", BindType.String)]
+        public string? BackgroundPositionX;
+
+        [Bind("background-position-y", BindType.String)]
+        public string? BackgroundPositionY;
+
+        [Bind("background-repeat", BindType.String)]
+        public string BackgroundRepeat = "repeat";
+
+        [Bind("background-size", BindType.String)]
+        public string BackgroundSize = "auto";
+
+        [Bind("background-url", BindType.String)]
+        public string? BackgroundUrl;
+
+        [Bind("border", BindType.String)]
+        public string? Border;
+
+        [Bind("border-bottom", BindType.String)]
+        public string? BorderBottom;
+
+        [Bind("border-left", BindType.String)]
+        public string? BorderLeft;
+
+        [Bind("border-radius", BindType.String)]
+        public string? BorderRadius;
+
+        [Bind("border-right", BindType.String)]
+        public string? BorderRight;
+
+        [Bind("border-top", BindType.String)]
+        public string? BorderTop;
+
+        [Bind("direction", BindType.String)]
+        public string Direction = "ltr";
+
+        [Bind("full-width", BindType.String)]
+        public string? FullWidth;
+
+        [Bind("padding", BindType.PixelsOrPercent)]
+        public string Padding = "20px 0";
+
+        [Bind("padding-bottom", BindType.PixelsOrPercent)]
+        public string? PaddingBottom;
+
+        [Bind("padding-left", BindType.PixelsOrPercent)]
+        public string? PaddingLeft;
+
+        [Bind("padding-right", BindType.PixelsOrPercent)]
+        public string? PaddingRight;
+
+        [Bind("padding-top", BindType.PixelsOrPercent)]
+        public string? PaddingTop;
+
+        [Bind("text-align", BindType.Align)]
+        public string TextAlign = "center";
+
+        [Bind("text-padding", BindType.PixelsOrPercent)]
+        public string TextPadding = "4px 4px 4px 0";
+
+        [Bind("css-class")]
+        public string? CssClass;
+
         public override void Render(IHtmlRenderer renderer, GlobalContext context)
         {
+            if (IsFullWidth())
+            {
+                RenderFullWidth(renderer, ref context);
+            }
+            else
+            {
+                RenderSimple(renderer, ref context);
+            }
+        }
+
+        private void RenderFullWidth(IHtmlRenderer renderer, ref GlobalContext context)
+        {
+            var hasBackground = HasBackground();
+
+            var tableElement = renderer.ElementStart("table")
```

Comment:
I Started to write the style next to it like // Style div.

Otherwise it is hard to understand later where the styles come from.